scripts/ci/image-counter-lambda-build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+apt-get update
+apt-get install zip
```

Comment:
why is this needed?

Reply:
We are running this CI script from inside a node docker image that doesn't have zip, and riffraff-artefact requires it. I personally think this is a bit cleaner than messing around with nvm stuff, as the image takes care of the node version by virtue of using `node:10`.

Reply:
I just saw this, and this is fixed by using node-riffraff-artifact